server/spike/adk/run_phase_a.sh

```
@@ -0,0 +1,67 @@
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR"
+
+# Activate venv
+if [ -d ".venv" ]; then
+    source .venv/bin/activate
+fi
+
+# Load API key from project .env if not already set
+if [ -z "${GOOGLE_API_KEY:-}" ]; then
+    if [ -f "../../../.env" ]; then
+        export GOOGLE_API_KEY="$(grep GOOGLE_AI_API_KEY ../../../.env | cut -d= -f2)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)